.github/workflows/test.yml

```diff
@@ -18,3 +18,34 @@ jobs:
         with:
           route: GET /repos/:owner/:repo/releases/latest
       - run: "echo latest release: '${{ steps.get_latest_release.outputs.data }}'"
+      # Create check run
+      - uses: ./
+        id: create_check_run
+        with:
+          route: POST /repos/:owner/:repo/check-runs
+          headers: '{"accept": "application/vnd.github.antiope-preview+json"}'
+          name: 'Test check run'
+          head_sha: ${{ github.sha }}
+          output: '{"title":"Test check run title","summary": "A summary of the test check run", "images": [{"alt": "Test image", "image_url": "https://octodex.github.com/images/jetpacktocat.png"}]}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      
+      # Parse steps.create_check_run.outputs.data, since it is a string
+      - id: parse_create_check_run
+        uses: gr2m/get-json-paths-action@v1.x
+        with:
+          json: ${{ steps.create_check_run.outputs.data }}
+          id: "id"
+          
+      # Update check run to completed, succesful status
+      - uses: ./
+        id: update_check_run
+        with:
+          route: PATCH /repos/:owner/:repo/check-runs/:check_run_id
+          check_run_id: ${{ steps.parse_create_check_run.outputs.id }}
```

Comment:
I think I'd move `check_run_id` below `mediaType`, so the order is

1. Route
2. Previews
3. Parameters